### PR TITLE
Adding parties to filing package

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding parties to filing package

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit test and 

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java [hunk 1]
```diff
@@ -12,28 +12,29 @@
...
-    @Mapping(target = "ldcxCourtClassCd", source = "filingPackage.court.courtClass")
-    @Mapping(target = "ldcxCourtLevelCd", source = "filingPackage.court.level")
-    @Mapping(target = "submittedToAgenId", source = "filingPackage.court.agencyId")
-    @Mapping(target = "submittedByAccountId", source = "accountId")
-    @Mapping(target = "submittedByClientId", source = "clientId")
-    @Mapping(target = "ldcxCourtDivisionCd", source = "filingPackage.court.division")
-    EfilingFilingPackage toEfilingFilingPackage(Submission submission);
+    @Mapping(target = "ldcxCourtClassCd", source = "submission.filingPackage.court.courtClass")
+    @Mapping(target = "ldcxCourtLevelCd", source = "submission.filingPackage.court.level")
+    @Mapping(target = "submittedToAgenId", source = "submission.filingPackage.court.agencyId")
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java:25]: should this be:
```suggestion
    @Mapping(target = "submittedToAgencyId", source = "submission.filingPackage.court.agencyId")
```

Comment (sdevalapurkar-bcgov) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java:25]: just two extra characters but makes it much easier to understand what that var is

Comment (alexjoybc) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java:25]: unfortunatly it's comming from the cso object, nothing we can do about it...

Comment (TayGov) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/mappers/EfilingFilingPackageMapper.java:25]: Yep those are not under our control.

**Pull request merged**